src/components/views/elements/FacePile.tsx

```diff
@@ -32,6 +32,10 @@ interface IProps extends Omit<HTMLAttributes<HTMLDivElement>, "onChange"> {
     onClick?: (e: ButtonEvent) => void | Promise<void>;
 }
 
+/**
+ * A component which displays a list of avatars in a row, with a tooltip showing the names of the users.
+ * Any additional props inherited from the div element are passed to the underlying AccessibleButton.
```

Comment:
Shouldn't this go on the `...props` ? or the `IProps` class?

Also, I don't think "inherited from the div element" is quite the right thing

```suggestion
 *
 * Any additional props, not named explicitly here, are passed to the underlying {@link AccessibleButton}.
```

Reply:
> Shouldn't this go on the ...props ? or the IProps class?

comments on `IProps` would not be shown by modern IDEs given they'd only be looking at the component so that'd be less useful IMO